EventsExpress/ClientApp/src/config.js

```diff
@@ -1,4 +1,7 @@
 ﻿export default {
   "FACEBOOK_CLIENT_ID": "2741444662848625",
-  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com"
+  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com",
+  "TWITTER_CALLBACK_URL": "https://localhost:5002/Authentication/TwitterLogin",
```

Comment:
This will not work properly on stage and prod, since it will not be redirected to localhost there. Consider using relative URL here.